etc/di.xml

```diff
@@ -53,6 +53,12 @@
     <preference for="Bolt\Boltpay\Api\Data\FeatureSwitchInterface" type="Bolt\Boltpay\Model\FeatureSwitch"/>
     <preference for="Bolt\Boltpay\Api\FeatureSwitchRepositoryInterface" type="Bolt\Boltpay\Model\FeatureSwitchRepository"/>
 
+    <!-- For debug endpoint -->
+    <preference for="Bolt\Boltpay\Api\DebugInterface" type="Bolt\Boltpay\Model\Api\Debug" />
+    <preference for="Bolt\Boltpay\Api\Data\DebugInfoInterface" type="Bolt\Boltpay\Model\Api\Data\DebugInfo" />
+    <preference for="Bolt\Boltpay\Api\Data\PluginVersionInterface" type="Bolt\Boltpay\Model\Api\Data\PluginVersion" />
+    <preference for="Bolt\Boltpay\Api\Data\BoltConfigSettingInterface" type="Bolt\Boltpay\Model\Api\Data\BoltConfigSetting" />
```

Comment:
Why make these di classes as opposed to a simple Php class?

Reply:
I actually just followed the same way as other places (eg: shipping methods endpoint) and thought it was the recommended way as Magento2 seems to really like DI.

@danac-gs @YevhenShBolt @matt-thomason @njfaries  What are your thoughts on these?

Reply:
Synced on Slack and decided to only leave DebugInterface.

Reply:
removed